geode-core/src/test/java/org/apache/geode/test/dunit/rules/ClusterStartupRule.java

```diff
@@ -50,12 +51,15 @@
 import org.apache.geode.test.junit.rules.ServerStarterRule;
 import org.apache.geode.test.junit.rules.VMProvider;
 
-
 /**
  * A rule to help you start locators and servers inside of a
  * <a href="https://cwiki.apache.org/confluence/display/GEODE/Distributed-Unit-Tests">DUnit
  * test</a>. This rule will start Servers and Locators inside of the four remote {@link VM}s created
  * by the DUnit framework.
+ *
+ * <p>
+ * If you use this Rule in a WAN or other test that uses more than getVM(0 through getVM(3), then
```

Comment:
probably not mention WAN test at all, just say, if your test uses more than the 4 default vms (vm0 to vm3), you will need to use the constructor with an integer param.

Reply:
Resolved